CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Latest Changes
+## 0.5.0
```

Comment:
```suggestion
## [0.5.0]
```

Need to follow the Keep A Changleog format: https://keepachangelog.com/en/1.1.0/

Reply:
Sorry.